backend/src/govsg/services/govsg.service.ts

```diff
@@ -139,6 +140,20 @@ const getLanguageCode = (language: string | undefined) => {
   return WhatsAppLanguages[key as keyof typeof WhatsAppLanguages]
 }
 
+const isUsingPrecallTemplate = async (govsgMessage: GovsgMessage) => {
```

Comment:
nit: Is this function name accurate? Having a passcode goes beyond just precall templates right? Perhaps `isTemplateWithPasscode` is better?

Reply:
For current use cases, the name is accurate. But sure, let me rename that.

Reply:
Done!